docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,151 @@
+ - Ensured the Windows Firewall has a rule configured to allow incoming connections on TCP Port 10933 which will allow the Octopus Server to talk to our new Tentacle.
+
+Using the Tentacle Manager stop the Octopus Tentacle service which was just installed on the first node and take the shared disk offline in Windows Disk Management. 
+
+Now go to the second tentacle server in the Active/Passive cluster and bring the same disk online, repeating the steps which were just performed on the first node to install the Octopus Tentacle service. Please keep the Octopus Tentacle service started and ensure that the Shared Storage is still mounted this time so that the SSL Certificate can be exported out of Octopus Tentacle.
+
+## Generate an Octopus Tentacle Private Key {#ClusteringTentacles-NewPrivateKey}
+
+Open `cmd` again and run these commands to generate a new Private Key from an Octopus Deploy Tentacle.
+```
+cd "C:\Program Files\Octopus Deploy\Tentacle\"
+Tentacle.exe new-certificate -e "TentacleClusterPrivateKey.txt"
+```
+
+*![](/docs/images/clustered-listening-tentacles/export-privatekey.jpg)*
+
+## Import the new Octopus Tentacle Private Key {#ClusteringTentacles-ImportPrivateKey}
+
+Now import the new private key into the server from which it was just generated.
+```
+Tentacle.exe import-certificate -instance "Tentacle" -f "TentacleClusterPrivateKey.txt"
+Tentacle.exe service --instance "Tentacle" --stop --start
+```
+
+Now on the second node, stop the Tentacle service and bring the shared storage offline the same way you have done before. This time go back to the first node in the cluster, bringing the Shared Storage drive back online and start the tentacle service. Copy the newly exported "TentacleClusterPrivateKey.txt" file to the first node, then follow the step above to import it into the first node to ensure both nodes in the cluster hold the same private key. 
+
+Once both Tentacles are installed and configured ensure that neither node has the Octopus Tentalce started and that the shared storage is brought offline.
+
+*![](/docs/images/clustered-listening-tentacles/import-privatekey.jpg)*
+
+## Configure A new Clustered Service {#ClusteringTentacles-NewCluster}
+
+Ensure each node that will be participating in the Tentacle Cluster is joined to the Active Directory Domain and has the **Failover Clustering** Feature installed in Windows.
+
+Open the **Failover Cluster Manager** console on one of the nodes. If there is no cluster configured yet, you can right click **Failover Cluster Manager** amd select **New Cluster**
+
+On the **Select Servers** page, enter the Fully Qualified Domain Name of each node that will be in this cluster.
+
+On the **Validation Warning** page, select "Yes, When I click Next, run configuration validation tests, and then return to the process of creating the cluster"
+
+The **Validate a Configuration Wizard** appears, select **Run all Tests** and select **Next**
```

Comment:
is `run all tests` a checkbox or a button? if its a checkbox, surround it with backticks.